src/test/config.sol

```diff
@@ -1636,5 +1636,65 @@ contract Config {
             calc_cut:     9900,
             lerp:         false
         });
+        afterSpell.collaterals["RWA008-A"] = CollateralValues({
```

Comment:
Might be helpful to move these adjacent to the other RWA types for easier discovery.

Reply:
Fixed

Reply:
Resolving this